src/pages/index.js

```diff
@@ -42,24 +124,44 @@ editFormValidator.enableValidation();
 const addFormValidator = new FormValidator(config, addNewCardFormEl);
 addFormValidator.enableValidation();
 
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: (initialCardData) => {
-      const card = createCard(initialCardData);
-      cardSection.prependItem(card);
-    },
-  },
-  ".cards__list"
+const editAvatarFormValidator = new FormValidator(
+  config,
+  profileEditAvatarFormEl
 );
-
-cardSection.renderItems();
+editAvatarFormValidator.enableValidation();
 
 const addNewCardForm = new PopupWithForm("#add-image-popup", (inputValues) => {
-  const cardEl = createCard(inputValues);
-  cardSection.prependItem(cardEl);
+  api.addNewCard(inputValues).then((responseDataObj) => {
```

Comment:
Needs correction: don't forget to notify the user about the loading process by changing the button text

Reply:
corrected